afterpay/src/main/kotlin/com/afterpay/android/model/Money.kt

```diff
@@ -1,9 +1,13 @@
 package com.afterpay.android.model
 
+import com.afterpay.android.internal.BigDecimalSerializer
+import com.afterpay.android.internal.CurrencySerializer
 import kotlinx.serialization.Serializable
+import java.math.BigDecimal
+import java.util.Currency
 
 @Serializable
 data class Money(
```

Comment:
Is this class used outside of the SDK?

Reply:
It is.

Reply:
I guess this would be a breaking change then? Not that that's going to be a problem, I think

Reply:
Yep, but I feel that the type-safety is worth it.

Reply:
I could add a new constructor and deprecate the original, but as you say, if no-one has adopted the SDK yet then it shouldn't matter.

Reply:
Yeah it's a good change. It might mean a version bump  :+1:

Reply:
Definitely.